### PR TITLE
docs(readme): update documentation for docs-readme OT

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
Just a little polish / rewording work on the docs-readme OT docs page.

### CODE REVIEW

File: docs/output-targets/docs-readme.md [hunk 1]
```diff
@@ -28,16 +33,28 @@ stencil build --docs
...
 This will cause the Stencil compiler to perform a one-time generation of README
-files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output target.
+files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output
+target.
+
+:::caution
+If you use the `--docs` flag and don't add the `docs-readme` output target to
+your Stencil configuration your documentation won't be automatically updated
+when you build your project and could get out of date.
+:::
```
Comment (alicewriteswrongs) [inline docs/output-targets/docs-readme.md:43]: not sure this is really a `caution`, maybe it's just a `note` ?

Comment (rwaskiewicz) [reply docs/output-targets/docs-readme.md:43]: Yeah, I'd say "note", but I could see it going either way

**Comment by rwaskiewicz** [comment]
@alicewriteswrongs I merged #1102, I think you may need to rebase/resolve conflict with the header

File: versioned_docs/version-v3.2/documentation-generation/docs-readme.md [hunk 2]
```diff
@@ -28,16 +33,28 @@ stencil build --docs
 ```
 
 This will cause the Stencil compiler to perform a one-time generation of README
-files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output target.
+files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output
```
Comment (tanner-reits) [inline versioned_docs/version-v3.2/documentation-generation/docs-readme.md:36]: Doesn't this just link to itself?

**Review by tanner-reits — APPROVED** [review]
Just one non-blocking question

File: versioned_docs/version-v3.2/documentation-generation/docs-readme.md (hunk 2 above)
Comment (alicewriteswrongs) [reply versioned_docs/version-v3.2/documentation-generation/docs-readme.md:36]: lol it definitely does! I'll get rid of that while we're in here.

**Pull request merged**